Cargo.toml

```diff
@@ -10,15 +10,16 @@ description = "Utility functions for generating Rust code from protobufs (using
 default = ["protobuf-codec"]
 protobuf-codec = ["protobuf-codegen", "protobuf"]
 grpcio-protobuf-codec = ["grpcio-compiler/protobuf-codec", "protobuf-codec"]
-prost-codec = ["grpcio-compiler/prost-codec", "syn", "quote", "bitflags", "prost-build"]
+prost-codec = ["syn", "quote", "prost-build"]
 grpcio-prost-codec = ["grpcio-compiler/prost-codec", "prost-codec"]
 
 [dependencies]
+lazy_static = "1.3"
 protobuf = { version = "2", optional = true }
 protobuf-codegen = { version = "2", optional = true }
 grpcio-compiler = { version = "0.5.0-alpha", default-features = false, optional = true }
 prost-build = { version = "0.5", optional = true }
 regex = { version = "1.1" }
 syn = { version = "0.15", features = ["full"], optional = true }
 quote = { version = "0.6", optional = true }
-bitflags = { version = "1.0.4", optional = true }
+bitflags = "1.0"
```

Comment:
Why was it optional?

Reply:
It used to be that GenOpts was only used in the Prost configuration, but now you can set the GenOpts on the Builder in either configuration but it only has an effect with Prost.

Reply:
Gotcha